encoding/rs/bundle_builder.go

```diff
@@ -0,0 +1,33 @@
+package rs
+
+import (
+	"fmt"
+	"github.com/Layr-Labs/eigenda/core"
+)
+
+// BuildChunksData creates a binary core.ChunksData object from the given proofs and coefficients.
+func BuildChunksData(
+	proofs [][]byte,
+	chunkLen int,
+	coefficients [][]byte) (*core.ChunksData, error) {
+
+	if len(proofs) != len(coefficients) {
```

Comment:
does chunkLen need to validate?

Reply:
`chunkLen` is the size of the `[]fr.Element` for each chunk. It's a required piece of data that gets serialized in the bundle format. It's size is independent of `len(proofs)` and `len(coefficients)`.